lib/rules/no-bare-strings.js

```diff
@@ -35,6 +35,8 @@ const GLOBAL_ATTRIBUTES = [
 
 const TAG_ATTRIBUTES = {
   input: ['placeholder'],
+  Input: ['placeholder'],
+  Textarea: ['placeholder'],
```

Comment:
Should we also check `@placeholder` for these two?

Reply:
Looks like we did support those named arguments, though now with https://github.com/emberjs/rfcs/blob/master/text/0707-modernize-built-in-components-2.md they are deprecated.

I'd like to check them also though..